### ISSUE TITLE
lorem().characters() method sometimes does not include digit.

### ISSUE THREAD

**Issue by swaroop-tata** [DiUS/java-faker#759]
**Describe the bug**
lorem().characters() method sometimes does not include digit.

**To Reproduce**
Run "lorem().characters(7,true,true)" this in a loop.  Observe that some times, the generated string will miss digit.

**Expected behavior**
A digit must include every time when includeDigit is used.

**Versions:**
 - OS: Windows
 - JDK 11
 - Faker Version 1.0.2

**Additional context**

**Comment by bodiam** [comment]
You'd best use https://github.com/datafaker-net/datafake instead, which is an active fork of this library.

**Comment by bodiam** [comment]
I don't think this is a bug though, it's an option to include numbers. Not always they will be included. If you want to force numbers, you can use something like bothify, regexify, or examplify.

**Comment by swaroop-tata** [comment]
Thank you for the response.

**Comment by snuyanzin** [comment]
In datafaker there is a special `Text` provider where it's possible to specify minimum number of occurrencies per symbol. I guess, even more it's possible to specify other symbols e.g. from non English abc, it will satisfy your case.
For more details https://github.com/datafaker-net/datafaker/blob/main/src/test/java/net/datafaker/providers/base/TextTest.java
